test-app/tests/utils/function/rendering-test.gts

```diff
@@ -7,7 +7,7 @@ import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
 import { setOwner } from '@ember/application';
 
-import { use, resource, resourceFactory } from 'ember-resources';
+import { cell, use, resource, resourceFactory } from 'ember-resources';
```

Comment:
Looks like this cell import is unused, probably leftover from investigation

Reply:
thanks, yeah, I need to fix linting on this repo :upside_down_face: 
there is a bug with prettier rn though, and I'm waiting for them to do another release